tests/ast/data/functions.test

```diff
@@ -664,7 +664,7 @@ df32 = df.select(avg(col("A")), avg(col("B")))
 
 df33 = df.select(corr(col("A"), col("B")))
 
```

Comment:
the encoding here used to be wrong. I.e., due to some logic `count(*)` got demoted to `count(1)`. Yet, `count(*)` is closer to original. Changed here to be more correct.